### PR TITLE
LPS-149266

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hey Dante, 

Thanks a lot for your great comment, it is very inspiring!
I have made some changes based on your comments, would you please help me have a look?
Thanks for your checking!
FYI, https://github.com/jeffwu0724/liferay-portal/pull/89

Best,
Jeff

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:sf

**Comment by jeffwu0724** [comment]
ci:test:environment

**Comment by jeffwu0724** [comment]
ci:test:environments

File: portal-impl/test/unit/com/liferay/portal/util/FileImplTest.java [hunk 1]
```diff
@@ -274,41 +277,33 @@ public void testUnzipZipSlipVulnerable() throws Exception {
...
-				return true;
-			}
+			Assert.assertEquals(
+				"Entry is outside of the target dir: ../../../../../../bad.txt",
+				logEntry.getMessage());
 
-			return false;
+			Assert.assertTrue(
+				Files.exists(destinationPath.resolve("good.txt")));
+			Assert.assertFalse(Files.exists(parentPath.resolve("evil.txt")));
```
Comment (dantewang) [inline portal-impl/test/unit/com/liferay/portal/util/FileImplTest.java:301]: @jeffwu0724 The file name is bad.txt or evil.txt? Also, please consider my new comment: https://github.com/jeffwu0724/liferay-portal/pull/89#discussion_r891850242

**Review by dantewang — CHANGES_REQUESTED** [review]

File: portal-impl/test/unit/com/liferay/portal/util/FileImplTest.java (hunk 1 above)
Comment (dantewang) [reply portal-impl/test/unit/com/liferay/portal/util/FileImplTest.java:301]: cc @tinatian

**Comment by jeffwu0724** [comment]
Hey @dantewang ,

Thanks for your advice! 
1. Sorry for the typo "evil.txt", and I have fixed it to "bad.txt".
2. After reading your comment in https://github.com/jeffwu0724/liferay-portal/pull/89#discussion_r891850242, and I tried again with `../bad.txt`, and without the fix, it will be extracted into the temp dir and the test can easily find it. 
Please let me know if there is still any place that need me to work on.
Thanks for checking!

Best
Jeff

**Comment by dantewang** [comment]
@jeffwu0724 Thanks! I'll continue to look into the test later today.

**Comment by jeffwu0724** [comment]
> @jeffwu0724 Thanks! I'll continue to look into the test later today.

@dantewang great，thanks a lot!😀

**Comment by dantewang** [comment]
See https://github.com/jeffwu0724/liferay-portal/pull/90

**Pull request closed without merging**